webpush/encrypt.go

```diff
@@ -270,7 +281,7 @@ func hkdf(salt, ikm, info []byte, length int) []byte {
 	mac = hmac.New(sha256.New, prk)
 	mac.Write(info)
 	mac.Write([]byte{1})
-	return mac.Sum(nil)[0:length]
+	return mac.Sum(nil)[0:length], nil
```

Comment:
True, missed the fact that it's SHA256. In which case I understand the issue even less now. What's the problem here when you know exactly the max length and you control all the callers to the function where length is hardcoded.

How can there be "If the length requested is more than 32 ..." in #14 when using SHA256?